decidim-dev/lib/decidim/dev/test/rspec_support/warden.rb

```diff
@@ -1,7 +1,23 @@
 # frozen_string_literal: true
 
+module Decidim
+  module WardenTestHelpers
+    include Warden::Test::Helpers
+
+    #
+    # Utility method to login in the middle of a test as a different user from
+    # the current one.
+    #
+    def relogin_as(user, scope:)
```

Comment:
Yes, you mean that I don't expose the option and just pass user directly? I did that so that the calls to `relogin_as` keep the same form as `login_as`. But I can remove it.

Regarding the parameter to `logout`, absolutely, nice catch. I'll fix it!